statsmodels/tsa/statespace/representation.py

```diff
@@ -237,19 +237,22 @@ class Representation(object):
     """
     transition = MatrixWrapper('transition', 'transition')
     r"""
-    (array) Transition matrix: :math:`T~(k\_states \times k\_states \times nobs)`
+    (array) Transition matrix:
+    :math:`T~(k\_states \times k\_states \times nobs)`
```

Comment:
I think this has to be on one line for Sphinx, not sure if they fixed it though.

Reply:
Other entries are on two lines, e.g. line 234-237 above:

```
     r"""
     (array) Observation covariance matrix:
     :math:`H~(k\_endog \times k\_endog \times nobs)`
     """
```

Reply:
Is there a conclusion on what the right thing to do is here?